src/server/utils/logging/logger.js

```diff
@@ -15,20 +15,38 @@
  */
 
 import { argv } from 'yargs';
-import pino from 'pino';
```

Comment:
is this exported both as the default and a named export?

Reply:
yes